### DISCUSSION TITLE
We Need a Supabase Mascot

### DISCUSSION THREAD

**Discussion by sandbox-apps** [supabase/supabase#2424] [General]
![image](https://user-images.githubusercontent.com/86398599/125952877-bfa507a0-e1cc-4a92-b05f-18edc25f74da.png)
![image](https://user-images.githubusercontent.com/86398599/125954020-4008fc0a-2ff3-42c3-a184-84362aae523c.png)
![image](https://user-images.githubusercontent.com/86398599/125954181-7e0576a1-9662-4558-bd8f-42c894e3636f.png)

**Comment by kiwicopple** [comment]
😄 

I think this one might be better as a Discussion. I'll transfer it now!

**Reply by sandbox-apps** [reply]
Dash (Flutter) will be happy to meet new friend :D
